### ISSUE TITLE
What is your least favorite thing about your favorite language?

### ISSUE THREAD

**Issue by tigt** [helenvholmes/ama#5]

**Comment by helenvholmes** [comment]
My least favorite thing about English is that people sometimes feel the need to tell me English is an ugly language but "it's good for technical jargon". I've studied more "traditionally beautiful" languages and all of them have the capacity to be ugly and the capacity to be beautiful. And I think we all have a special cranny for our native tongue, so I'm not sure why people feel the need to tell me this. I make it a point not to call other languages ugly, anyway.

I also find it very challenging, even as a native speaker, to stumble upon new words and mispronounce them. In fact most of my childhood embarrassments have arisen from trying to use words I'd only read and not checked the pronunciation for and having my mother correct me. (Why must she be the walking dictionary that she is.) These have included:

- omnipotent (I tried to say "om-knee-poe-tent")
- specific/Pacific (actually these I would just swap and people would tease me for it)
- vicissitude (actually I don't even know how I mispronounced this one, I think s noises just fell out of my mouth)

For JavaScript... my least favorite part are the detractors in our community. It's a similar thing to above: I don't understand the vitriol for something that's so versatile. It's a nice language. We've done some fun stuff with it. Other languages are cool too.
